tempto-runner/src/main/java/io/prestosql/tempto/runner/TemptoRunner.java

```diff
@@ -120,8 +121,9 @@ private XmlSuite getXmlSuite()
         testSuite.setFileName("tempto-tests");
         XmlTest test = new XmlTest(testSuite);
         test.setName("all");
-        XmlPackage testPackage = new XmlPackage(options.getTestsPackage());
-        List<XmlPackage> testPackages = newArrayList(testPackage);
+        List<XmlPackage> testPackages = options.getTestsPackage().stream()
+                .map(XmlPackage::new)
+                .collect(toImmutableList());
```

Comment:
verify this is not an empty list?

Reply:
It does not look like a mistake if test package is omitted from the point of view of testng.